detectors/node/opentelemetry-resource-detector-instana/README.md

```diff
@@ -40,10 +40,11 @@ const sdk = new NodeSDK({
    resource: globalResource,
 });
 
-(async () => {
-   await sdk.detectResources();
+sdk.start()
 
-   await sdk.start();
+(async () => {
+   const resource = sdk["_resource"];
+   await resource.waitForAsyncAttributes?.();
 }());
```

Comment:
I think this is no longer required at all right? The SDK should transparently handle the Async resources

Reply:
Yeah I wasn't sure earlier. In general it might be good to show both cases? 
#1 Start SDK & load everything in the background and do not care. 
#2 Show how to wait till Otel has fully finished bootstrapping.